### PR TITLE
Merge dev staging

### PR DESCRIPTION
**Pull request by ravinderwalkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/mobile-home/mobile-home.component.scss [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+                border-radius: 5px !important;
+            }
+            .input-group-addon{
+                background-color: transparent;
+                border-radius: 0;
+                padding-left: 20px;
+                line-height: 1.2;
+                height: auto;
+                padding-right: 5px;
+                color: #808080;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/mobile-home/mobile-home.component.scss:195]: use color var name

File: apps/web-giddh/src/app/mobile-home/mobile-home.component.scss [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+            height: 36px;
+        }
+    }
+    .account-search{
+        margin: 0;
+        padding: 15px 0 15px 10px;
+        width: calc(100% - 36px);
+        position: relative;  
+        .input-group{
+            background-color: #fff;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/mobile-home/mobile-home.component.scss:180]: use color var name

File: apps/web-giddh/src/app/mobile-home/mobile-home.component.scss [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+                    position: relative;
+                    top: 1px;
+                }
+            }
+            .close-addon{
+                border-radius: 0;
+                padding-left: 0;
+                line-height: 1.2;
+                height: auto;
+                color: #808080;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/mobile-home/mobile-home.component.scss:211]: use color var name

File: apps/web-giddh/src/app/settings/settings.component.css [hunk 4]
```diff
@@ -1,3 +1,18 @@
...
+    .setting-sidebar-wrapper{
+        z-index: 999;
+        width: 100%;
+        height: 100%;
+        display: block;
+        background-color: rgba(0,0,0,0.5);
+        position: absolute;
+        top: 0; 
+    }
+    .ml-215{
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/settings/settings.component.css:15]: remove this css & use for this class ml-0

File: apps/web-giddh/src/app/settings/settings.component.css [hunk 5]
```diff
@@ -1,3 +1,18 @@
...
+    padding:0 15px 15px;
+    margin-top: 15px;
+}
+@media(max-width:767px){
+    .setting-sidebar-wrapper{
+        z-index: 999;
+        width: 100%;
+        height: 100%;
+        display: block;
+        background-color: rgba(0,0,0,0.5);
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/settings/settings.component.css:11]: use var color name

File: apps/web-giddh/src/app/settings/settings.component.html [hunk 6]
```diff
@@ -1,85 +1,121 @@
...
+                </div>
+                <div class="setting-container" [hidden]="activeTab !== 'discount'">
+                    <setting-discount></setting-discount>
+                </div>
+                <div class="setting-container" [hidden]="activeTab !== 'warehouse'">
+                    <setting-warehouse></setting-warehouse>
+                </div>
+                <!-- <tabset #staticTabs>
+                    <tab heading="Taxes" [active]="activeTab === 'taxes'" (select)="tabChanged('taxes')">
+                        <div class="setting-container">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/settings/settings.component.html:52]: Remove commented code if not usable

File: apps/web-giddh/src/app/shared/header/components/aside-help-support/aside-help-support.component.scss [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+    }
+    .contact-block{
+        text-align: center;
+        font-size: 20px;
+        line-height: 1.5;
+    }
+    }
+}
+.aside-pane-form {
+    background-color: #fff;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/shared/header/components/aside-help-support/aside-help-support.component.scss:140]: use var color name

File: apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.scss [hunk 8]
```diff
@@ -0,0 +1,262 @@
...
+            i{
+                top: 25px;
+                left: 30px;
+                color: gray;
+            }
+        }
+    }
+}
+.aside-pane-form {
+    background-color: #fff;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.scss:105]: use var color name

File: apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.scss [hunk 9]
```diff
@@ -0,0 +1,262 @@
...
+    width: 215px;
+    left: 0;
+    z-index: 999999;
+    width: 270px;
+    .back-to-main-slider{
+        background-color: $midnight-blue;
+        a{
+            display: flex;
+            align-items: center;
+            color: #fff;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.scss:204]: use color var name

File: apps/web-giddh/src/app/mobile-home/mobile-home.component.html [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+                <div class="wrap-group-account-input">
+                    <span class="item-search-wrapper" *ngFor="let item of listOfSelectedGroups; let idx = index">
+                        <span>
+                            <span class="item-search-name">{{item.name}}</span>
+                            <ng-container>
+                                    <span>&nbsp;❯&nbsp;</span>
+                            </ng-container>
+                        </span>
+                    </span>
+                    <input type="text" #searchElement class="form-control" [(ngModel)]="searchString" placeholder="Search" aria-describedby="" (keyup)="initSearch($event, $event.target.value)" (keydown)="handleKeydown($event)">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/mobile-home/mobile-home.component.html:20]: aria-describedby="" blank if not usable pls remove

File: apps/web-giddh/src/app/login/token-verify.component.ts [hunk 11]
```diff
@@ -32,14 +32,23 @@ export class TokenVerifyComponent implements OnInit, OnDestroy {
     public ngOnInit() {
         this.generalService.storeUtmParameters(this.route.snapshot.queryParams);
 
-        // this.token = this.route.snapshot.queryParams['token'];
-        this.token = this.generalService.getCookie('token');
-        if (this.token) {
+        // this.token = this.generalService.getCookie('token');
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/login/token-verify.component.ts:35]: remove commented code

File: apps/web-giddh/src/app/settings/settings.component.html (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/settings/settings.component.html:52]: we can't for now

File: apps/web-giddh/src/app/mobile-home/mobile-home.component.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/mobile-home/mobile-home.component.scss]: Resolved

File: apps/web-giddh/src/app/mobile-home/mobile-home.component.scss (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/mobile-home/mobile-home.component.scss]: Resolved

File: apps/web-giddh/src/app/settings/settings.component.css (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/settings/settings.component.css]: Resolved

File: apps/web-giddh/src/app/settings/settings.component.css (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/settings/settings.component.css:11]: Ignoring for now, i need to discuss.

File: apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.scss (hunk 8 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.scss]: Resolved

File: apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.scss (hunk 9 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.scss]: Resolved

File: apps/web-giddh/src/app/mobile-home/mobile-home.component.html (hunk 10 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/mobile-home/mobile-home.component.html]: Resolved

**Pull request merged**